Misc/NEWS.d/next/Library/2021-10-25-01-22-49.bpo-45429.VaEyN9.rst

```diff
@@ -0,0 +1,3 @@
+On Windows, :func:`time.sleep` now uses a waitable timer which supports
+high-resolution timers without increasing the timer frequency if possible.
```

Comment:
I suggest to omit "without increasing the timer frequency if possible":

```suggestion
high-resolution timers.
```

(same change where you copied this doc)

Reply:
Maybe you can explain that in Python 3.10, the best resolution was 1 ms, whereas it's now better (smaller) than 1 ms.